pkg/services/rpcsrv/server.go

```diff
@@ -2895,6 +2898,13 @@ func (s *Server) subscribe(reqParams params.Params, sub *subscriber) (any, *neor
 			flt := new(neorpc.ExecutionFilter)
 			err = jd.Decode(flt)
 			filter = *flt
+		case neorpc.MempoolEventID:
+			if !s.config.MempoolSubscriptionsEnabled {
+				return nil, neorpc.NewInternalServerError("MempoolSubscriptionsEnabled is set to false")
```

Comment:
Previous version was good, it's just about the fact that the message started with capital letter. Let's use `mempool subscriptions are disabled`.